features/fetching_articles_from_news_api.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Website should contain news from API
+  As a dev team
```

Comment:
Please clarify what kind of stakeholder “dev team” is.

Reply:
Will change this.